tutorials/physics/physics_introduction.rst

```diff
@@ -95,6 +95,8 @@ it will typically be equal to ``0.01666...`` (but not always, see below).
     physics calculations, so that the game behaves correctly if you change the
     physics update rate or if the player's device can't keep up.
 
+:: _physics_introduction_collision_layers_and_masks:
```

Comment:
The correct rST syntax is:

```suggestion
.. _doc_physics_introduction_collision_layers_and_masks:
```

Also, make sure to always prefix references with `doc_`.

Reply:
Thanks for the help @Calinou - I'll remove the classes changes.
On this point though, I could not find any reference in the documentation guidelines on a naming convention for links. I'm new to Sphinx, but from what I can see in their documentation it appears that doc_ is a Godot convention, but I can't see it called out anywhere in Godot's "How to contribute" section.